src/components/SuggestionsList/SuggestionsList.tsx

```diff
@@ -68,16 +70,16 @@ export default class SuggestionsList extends React.PureComponent<SuggestionsList
       <div className={join(['y-suggestions-list', 'y-hc-border', className, classNames.root])}>
         {hasResults && (
           <LayoutList direction="vertical">
-            {groupedItems.map((group: SuggestionItemGroupProps) => (
+            {groupedItems.map((group: SuggestionItemGroup) => (
               <LayoutListItem key={group.title}>
                 <LayoutList direction="vertical">
                   {group.items.map((item: SuggestionItem) => (
```

Comment:
same here-  i know you didnt add this but the can this be pulled out into a this.getItem or whatever?

Reply:
👍

Reply:
done